test/static.test.js

```diff
@@ -545,6 +549,20 @@ t.test('sendFile', t => {
         genericResponseChecks(t, response)
       })
     })
+
+    t.test('reply.sendFile() with rootPath', t => {
+      t.plan(3 + GENERIC_RESPONSE_CHECK_COUNT)
+      simple.concat({
+        method: 'GET',
+        url: 'http://localhost:' + fastify.server.address().port + '/root/path/override/test',
+        followRedirect: false
+      }, (err, response, body) => {
+        t.error(err)
+        t.strictEqual(response.statusCode, 200)
+        t.strictEqual(body.toString(), deepContent)
+        genericResponseChecks(t, response)
```

Comment:
here, right after this response check, could you add another request to sibling endpoint without using the rootPath in order to check that they don't conflict each other?